github/teams_test.go

```diff
@@ -1745,3 +1746,359 @@ func TestIDPGroup_Marshal(t *testing.T) {
 
 	testJSONMarshal(t, u, want)
 }
+
+func TestTeamsService_GetExternalGroup(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+
+	mux.HandleFunc("/orgs/o/external-group/123", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		fmt.Fprint(w, `{
+			"group_id": 123,
+			"group_name": "Octocat admins",
+			"updated_at": "2021-01-24T11:31:04-06:00",
+			"teams": [
+				{
+					"team_id": 1,
+					"team_name": "team-test"
+				},
+				{
+					"team_id": 2,
+					"team_name": "team-test2"
+				}
+			],
+			"members": [
+				{
+					"member_id": 1,
+					"member_login": "mona-lisa_eocsaxrs",
+					"member_name": "Mona Lisa",
+					"member_email": "mona_lisa@github.com"
+				},
+				{
+					"member_id": 2,
+					"member_login": "octo-lisa_eocsaxrs",
+					"member_name": "Octo Lisa",
+					"member_email": "octo_lisa@github.com"
+				}
+			]
+		}`)
+	})
+
+	ctx := context.Background()
+	externalGroup, _, err := client.Teams.GetExternalGroup(ctx, "o", 123)
+	if err != nil {
+		t.Errorf("Teams.GetExternalGroup returned error: %v", err)
+	}
+
+	updatedAt, _ := time.Parse(time.RFC3339, "2021-01-24T11:31:04-06:00")
```

Comment:
Here, and everywhere else you use `time.Parse`, please change this to use the already-existing `referenceTime` value created in: `github/timestamp_test.go`.